code/modules/language/spanish.dm

```diff
@@ -0,0 +1,18 @@
+/datum/language/spanish
+	name = "Old Earth Spanish"
+	desc = "A less common Earth language that's survived the ages, thanks to being so widespread for its time."
+	speech_verb = "says"
+	whisper_verb = "whispers"
+	key = "e"
+	flags = TONGUELESS_SPEECH | LANGUAGE_HIDE_ICON_IF_UNDERSTOOD
+	default_priority = 100
+
+	icon_state = "galcom"
```

Comment:
Sombrero? it needs to be uh defined whether spanish applies to south/middle americans or to spanish europeans.